crates/turborepo-scm/src/git.rs

```diff
@@ -94,11 +137,72 @@ impl Git {
         Ok(output.trim().to_owned())
     }
 
+    /// for GitHub Actions environment variables, see: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables
+    pub fn get_github_base_ref() -> Option<String> {
+        // make sure we're running in a CI environment
+        if env::var("CI").ok()?.as_str() != "true" {
```

Comment:
I'd consider using the `turborepo_ci` package, specifically you can detect the `Vendor` and double check that it's indeed GitHub

Reply:
great suggestion! didn't know about that crate - https://github.com/vercel/turborepo/pull/9169/commits/da8112e4d1667e9af99164dc34b4533b5a09923b I also added a predicate helper :)